vkubelet/vkubelet.go

```diff
@@ -136,6 +139,7 @@ func (s *Server) registerNode() error {
 			Capacity:    s.provider.Capacity(),
 			Allocatable: s.provider.Capacity(),
 			Conditions:  s.provider.NodeConditions(),
+			Addresses:   s.provider.NodeAddresses(),
```

Comment:
guess you also need to set the Status.DaemonEndpoints.KubeletEndpoint.Port to "KUBELET_PORT" here.

Reply:
done